src/display/ui_state.rs

```diff
@@ -134,20 +134,23 @@ impl UIState {
                 total_bytes_downloaded += connection_info.total_bytes_downloaded;
                 total_bytes_uploaded += connection_info.total_bytes_uploaded;
 
+                let data_for_process: &mut NetworkData;
+
                 if let Some(process_name) =
                     UIState::get_proc_name(&connections_to_procs, &connection.local_socket)
                 {
-                    let data_for_process = processes.entry(process_name.clone()).or_default();
-                    data_for_process.total_bytes_downloaded +=
-                        connection_info.total_bytes_downloaded;
-                    data_for_process.total_bytes_uploaded += connection_info.total_bytes_uploaded;
-                    if !connection_previously_seen {
-                        data_for_process.connection_count += 1;
-                    }
+                    data_for_process = processes.entry(process_name.clone()).or_default();
                     connection_data.process_name = process_name.clone();
                 } else {
+                    data_for_process = processes.entry(String::from("<UNKNOWN>")).or_default();
                     connection_data.process_name = String::from("<UNKNOWN>");
```

Comment:
I would recommend doing something like `processes.entry(connection_data.process_name.clone())` to avoid retyping the string `<UNKNOWN>`.